webpack.config.js

```diff
@@ -3,6 +3,9 @@ const path = require('path');
 module.exports = {
   entry: './client/index.js',
   mode: 'development',
+  resolve: {
```

Comment:
I ended up also changing the webpack config to better support the a11y add-on, but I did not have these lines. Good catch!